Moonscraper Chart Editor/Assets/Scripts/Game/Charts/IO/Midi/MidReader.cs

```diff
@@ -521,58 +516,20 @@ private static void ReadNotes(IList<MidiEvent> track, Song song, Song.Instrument
                     SongObjectHelper.GetRange(notes, tick, tick + endPos, out index, out length);
                     for (int k = index; k < index + length; ++k)
                     {
-                        notes[k].guitarFret = Note.GuitarFret.Open;
-
-                        if (gameMode == Chart.GameMode.Drums)
-                            notes[k].guitarFret = LoadDrumNoteToGuitarNote(notes[k].guitarFret);
+                        if (gameMode != Chart.GameMode.Drums)
+                            notes[k].guitarFret = Note.GuitarFret.Open;
```

Comment:
Under what cases are we running into sysex events in drums? Do sysex events mean something else on drums that needs separate handling? Comment explanation needed.

Reply:
Under normal circumstances there shouldn't be SysEx events on drums, but in the case that there are some then they certainly shouldn't be interpreted as open notes. One case where there might be is when a Phase Shift `PART REAL_DRUMS_PS` track is copied over to the regular `PART DRUMS` track since they're entirely compatible; PS's Real Drums uses SysEx events to mark things, but these events don't mean anything outside of that mode currently.

Reply:
This behavior would be a bit more clear with more proper SysEx parsing, I have something for that stashed and I could get it in another PR once this is merged in.

Reply:
That's fine and, but I'm saying that this explanation needs to be documented in the code as a comment.

Reply:
Ah alright, will do so.

Reply:
Took a moment to sneak another SysEx fix in there (open note SysEx on GHL, while not typical, wouldn't set the note correctly), and also excluded unrecognized tracks since we don't know what's valid on those.